src/BenchmarkDotNet/Toolchains/DotNetCli/DotNetCliCommand.cs

```diff
@@ -38,10 +40,11 @@ public class DotNetCliCommand
             BuildPartition = buildPartition;
             EnvironmentVariables = environmentVariables;
             Timeout = timeout;
+            this.LogOutput = logOutput;
```

Comment:
nit: there is no need for `this` (casing difference)

```suggestion
            LogOutput = logOutput;
```